requirements.txt

```diff
@@ -3,5 +3,5 @@ djangorestframework==3.0.5
 httplib2==0.9.2
```

Comment:
Should this library change be mentioned in changelog?

Reply:
done